Sources/NIO/Linux.swift

```diff
@@ -71,12 +71,12 @@ internal enum Epoll {
     public static let EPOLL_CTL_DEL: CInt = numericCast(CNIOLinux.EPOLL_CTL_DEL)
 
     #if os(Android)
-    public static let EPOLLIN: CUnsignedInt = numericCast(CNIOLinux.EPOLLIN)
-    public static let EPOLLOUT: CUnsignedInt = numericCast(CNIOLinux.EPOLLOUT)
-    public static let EPOLLERR: CUnsignedInt = numericCast(CNIOLinux.EPOLLERR)
-    public static let EPOLLRDHUP: CUnsignedInt = numericCast(CNIOLinux.EPOLLRDHUP)
-    public static let EPOLLHUP: CUnsignedInt = numericCast(CNIOLinux.EPOLLHUP)
-    public static let EPOLLET: CUnsignedInt = numericCast(CNIOLinux.EPOLLET)
+    public static let EPOLLIN: CUnsignedInt = 1 //numericCast(CNIOLinux.EPOLLIN)
```

Comment:
ClangImporter doesn't register these constants, apparently [because of the empty `__force` macro](https://android.googlesource.com/platform/bionic/+/refs/heads/master/libc/kernel/uapi/linux/eventpoll.h#28) as deleting that word from that header gets it to work:
```
swift-nio/Sources/NIO/Linux.swift:74:59: error: module 'CNIOLinux' has no member named 'EPOLLIN'
```
Hacked in the numeric values for now, could try something else if anyone knows any other way.